schemas/JSON/manifests/v1.2.0/manifest.locale.1.2.0.json

```
@@ -35,6 +35,21 @@
           "description": "The agreement URL."
         }
       }
+    },
+    "Documentation": {
+      "type": "object",
+      "properties": {
+        "DocumentLabel": {
+          "type": [ "string", "null" ],
+          "minLength": 1,
+          "maxLength": 100,
+          "description": "The description of the documentation for providing software guides such as manuals and troubleshooting URLs."
+        },
+        "DocumentUrl": {
+          "$ref": "#/definitions/Url",
+          "description": "The documentation URL."          
+        }
+      }

```

Documentation shouldn't be able to contain empty objects in the array
```suggestion
      },
      "required": [
        "DocumentLabel",
        "DocumentUrl"
      ]
```